nanomind-connector/mo-consumers/src/main/java/esa/mo/nanomind/impl/util/NanomindServicesConsumer.java

```diff
@@ -42,6 +42,8 @@
 public class NanomindServicesConsumer {
     private static final Logger LOGGER = Logger.getLogger(NanomindServicesConsumer.class.getName());
 
+    private static NanomindServicesConsumer instance;
```

Comment:
I chose a singleton pattern here, as the class sets some system properties, which should be probably only set once, right?
Though it also has methods to set services (never used), I believe they still should be shared - that is, a singleton is acceptable here?
The change was needed as this class is very OPSSAT specific, but was used in the adapters (gps, power) constructors as argument. Problem then arouse within nanosat branch when classes are created using reflection.